src/translations/strings.yaml

```
@@ -928,6 +928,20 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+helpSheets: 
+  dnsTitle:
+    value: Custom DNS settings
+    comment: Title label for the custom dns help sheet
+  dnsHeader: 
+    value: What is custom DNS?
```

```suggestion
    value: What is a custom DNS?
```

Sounds off without articles?

---

@traceyvantyghem

---

Yes, agree with Flod: "What is a custom DNS?" (Sorry about the delayed reply)